critiquebrainz/frontend/external/musicbrainz.py

```diff
@@ -73,6 +73,27 @@ def browse_release_groups(artist_id=None, release_types=None, limit=None, offset
     return release_groups
 
 
+def browse_releases(artist_id=None, release_types=None, limit=None, offset=None, release_group=None, includes=None):
+    """Get all the releases by a certain artist.
+    You need to provide an artist's MusicBrainz ID or the Release Group's MusicBrainz ID
+    """
+    if release_types is None:
+        release_types = []
+    key = cache.gen_key(artist_id, limit, offset, *release_types)
```

Comment:
There is a problem with key generation here. You must include all arguments (I don't see `release_group` and `includes `) otherwise things that you don't expect will come out. I just had this happen on one of the pages.

I opened `/release-group/73202fad-3839-423c-958a-9d036c1089e2` and the data got cached. Then I opened `/release-group/885181a4-9133-485f-bfce-fddc1168a3c1` and saw this:
<img width="283" alt="screenshot 2016-12-06 18 39 19" src="https://cloud.githubusercontent.com/assets/460525/20936604/7e4b3420-bbe3-11e6-883a-c83170fda6fa.png">